File: Java/Lista2/Exerc20.java
package Lista2;

import javax.swing.JOptionPane;

public class Exerc20 {

	public static void main(String[] args) {
		// Escrever um algoritmo que leia o nome de um vendedor, o seu salário fixo
		// e o total de vendas efetuadas por ele no mês (em dinheiro). Sabendo que este
		// vendedor ganha 15% de comissão sobre suas vendas efetuadas, informar ao final
		// do programa o seu nome, o salário fixo e salário no final do mês. 
		
		String vendedor;
		double salarioFixo, vendas, salario;
		vendedor = JOptionPane.showInputDialog(null,"Informe o nome do vendedor: ");
		vendas = Double.parseDouble(JOptionPane.showInputDialog(null,"Informe as vendas efetuadas no mês: "));
		salarioFixo = Double.parseDouble(JOptionPane.showInputDialog(null,"Informe o valor do salário fixo: "));
		
		salario = salarioFixo + 0.15 * vendas;
		
		JOptionPane.showMessageDialog(null,"O salário no final do mês é R$ "+salario+".");		
	}

}

File: Python/Lista2/exerc20.py
# 20) Escrever um algoritmo que leia o nome de um vendedor,
# o seu salário fixo e o total de vendas efetuadas por ele
# no mês (em dinheiro). Sabendo que este vendedor ganha 15% de comissão
# sobre suas vendas efetuadas, informar ao final do programa o seu nome,
# o salário fixo e salário no final do mês.

vendedor = input("Informe o nome do vendedor: ")
fixo = float(input("Informe o salário fixo do vendedor: "))
vendas = float(input("Informe o total de vendas efetuadas: "))
salario = fixo + 0.15*vendas
print("Vendedor: ",vendedor)
print("Salário fixo: R$",fixo)
print("Salário total: R$",salario)
